app/src/main/kotlin/nl/q42/template/logging/CrashlyticsAntilog.kt

```diff
@@ -0,0 +1,40 @@
+package nl.q42.template.logging
+
+import com.google.firebase.crashlytics.ktx.crashlytics
+import com.google.firebase.ktx.Firebase
+import io.github.aakira.napier.Antilog
+import io.github.aakira.napier.DebugAntilog
+import io.github.aakira.napier.LogLevel
+
+/** A value suitable for Crashlytics */
+private const val MAX_CHARS_IN_LOG = 1200
+
+class CrashlyticsAntilog : Antilog() {
+
+    private val logcatAntilog = DebugAntilog()
+
+    override fun performLog(
+        priority: LogLevel,
+        tag: String?,
+        throwable: Throwable?,
+        message: String?
+    ) {
+        // also send to logcat
+        logcatAntilog.log(priority, tag, throwable, message)
+
+        val limitedMessage = message?.take(MAX_CHARS_IN_LOG) // to avoid OutOfMemoryError's
+
+
+        if ((priority == LogLevel.INFO || priority == LogLevel.DEBUG || priority == LogLevel.WARNING) && limitedMessage != null) {
+            val errorMessage = throwable?.let {
+                " with error: ${throwable}: ${throwable.message}".take(MAX_CHARS_IN_LOG)
+            } ?: ""
+            Firebase.crashlytics.log(limitedMessage + errorMessage)
```

Comment:
not a big thing, but I believe you could end up with MAX_CHARS_IN_LOG * 2 here, but that's no problem I think

Reply:
Yeah, thanks for bringing it up. I fear that using `take(MAX_CHARS_IN_MESSAGE - limitedMessage.length) would lead to a completely truncated exception message, so instead I documented this behavior as  intentional